industrial_ci/src/env.sh

```diff
@@ -35,6 +33,24 @@ if [ "$USE_DEB" ]; then  # USE_DEB is deprecated. See https://github.com/ros-ind
 fi
 if [ ! "$UPSTREAM_WORKSPACE" ]; then export UPSTREAM_WORKSPACE="debian"; fi
 
+# If not specified, use ROS Shadow repository http://wiki.ros.org/ShadowRepository
```

Comment:
Oope, `ros` is already an option. Then I don't know if there's any other binary repository worth being added here.
So +1 with as is.